### ISSUE TITLE
brotli4j 1.10.0 class files and Java 8

### ISSUE THREAD

**Issue by sullis** [hyperxpro/Brotli4j#88]
I am unable to use brotli4j version 1.10.0 with Java 8:

```

bad class file: /root/.gradle/caches/modules-2/files-2.1/com.aayushatharva.brotli4j/brotli4j/1.10.0/44597bbb51f0481baa9ba6ba081d38694394c698/brotli4j-1.10.0.jar(/com/aayushatharva/brotli4j/decoder/DecoderJNI.class)
[119](https://github.com/Netflix/zuul/actions/runs/4227072795/jobs/7341257524#step:4:120)
    class file has wrong version 61.0, should be 55.0

```

The previous library version (1.9.0) works fine.

**Comment by hyperxpro** [comment]
Lol no way.

**Comment by hyperxpro** [comment]
I'm doomed. Gonna do emergency release now 😂

**Comment by hcross** [comment]
yup confirmed that "bug" on JDK11 target project :)
```
[ERROR]   bad class file: /Users/hoanicross/.m2/repository/com/aayushatharva/brotli4j/brotli4j/1.10.0/brotli4j-1.10.0.jar(/com/aayushatharva/brotli4j/decoder/Decoder.class)
[ERROR]     class file has wrong version 61.0, should be 55.0
[ERROR]     Please remove or make sure it appears in the correct subdirectory of the classpath.
```

**Comment by hyperxpro** [comment]
Fixed in 1.11.0
